src/pages/App.tsx

```diff
@@ -156,7 +156,7 @@ export default function App() {
             <Suspense fallback={<Loader />}>
               {isLoaded ? (
                 <Routes>
-                  {exploreFlag === ExploreVariant.Enabled && (
+                  {tokensFlag === TokensVariant.Enabled && (
                     <>
                       <Route path="/tokens" element={<Explore />} />
```

Comment:
should we change the component name too here

Reply:
yes separately